planner/cascades/testdata/stringer_suite_in.json

```diff
@@ -22,6 +22,8 @@
       "select a from t where b > 1 order by c",
       // Union ALL.
       "select avg(a) from t union all select avg(b) from t",
+      // Index Lookup Read.
+      "select b, f, g from t where t.b = 2 and t.f = 1 order by t.f, t.g",
```

Comment:
index hints is more understandable than `order by` clause for developers who are not familiar with optimizer internals.

Reply:
But cascades planner doesn't support hint now.

Reply:
Maybe we can change it in the future.